Bofors / Kargil Re-Imagined

BlueHNT opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Both of the guns have the potential to be good, until now they were an afterthought, but with some minor playing around with the stats - I made sure that they have the potential to be useful as a long-range artillery with tighter swab.

- **Kargil** being a single fire Cyrexa can't stand up to it's brother, but it packs a bigger punch than it with the single shot that is does have.
- **Bofors** being the biggest of the 3 has the potential to wreck, not enough to be comparable to let's say a Charon, but it can hold it's own from the testing I've done.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
big gun do big damage now

## How to test
<!-- Describe the way it can be tested -->
shoot gun, gun strong

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
- [x] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- tweak: Kargil and Bofors now pack a real punch, while keeping a considerable delay between the shots.
- add: Adds guidebook entries for Kargil/Bofors/Smerchs

BlueHNT committed:
higher potential, small area

Redrover1760 commented:
Changelog please

also increase the range on bofers if they're still like 600m or so

BlueHNT commented:
> Changelog please
> 
> also increase the range on bofers if they're still like 600m or so

ah, didn't even notice that

BlueHNT committed:
decreases range from 2.615km to 1.575km

BlueHNT committed:
Updates guidelines

BlueHNT committed:
WE LOVE GUIDELINES

BlueHNT committed:
Merge branch 'main' of https://github.com/Monolith-Station/Monolith into Shipguns-v7-6

BlueHNT commented:
@Redrover1760 updated :cl: and added missing guidelines, updated old stuff while I was at it.

Redrover1760 commented:
> @Redrover1760 updated 🆑 and added missing guidelines, updated old stuff while I was at it.

its easy to forget but none of our ships are even capital ships so there's no way cyrexas should ever be "super-capital"

marauders also don't cause fires so they don't 'ignite' ships, so just say they melt ships or something

BlueHNT committed:
20 tons of inconsistencies

BlueHNT commented:
> > @Redrover1760 updated 🆑 and added missing guidelines, updated old stuff while I was at it.
> 
> its easy to forget but none of our ships are even capital ships so there's no way cyrexas should ever be "super-capital"

I kno, they were a hold in.

> marauders also don't cause fires so they don't 'ignite' ships, so just say they melt ships or something

But we love flavor-text :(

BlueHNT committed:
marauder melts

BlueHNT committed:
updates descriptions to reflect designations

BlueHNT committed:
Merge branch 'main' of https://github.com/Monolith-Station/Monolith into Shipguns-v7-6

monolith8319 committed:
chore: automatically update REUSE headers

monolith8319 committed:
chore: automatically update REUSE headers

BlueHNT commented:
[5eda114](https://github.com/Monolith-Station/Monolith/pull/913/commits/5eda1146ff9feb7ae3c908b7e5c4527ae30f1174) fuck off robot.
